content/about.md

```diff
@@ -4,24 +4,38 @@ date: 2019-02-16T13:56:52+01:00
 type: static
 ---
 
-# Maintainers and Contributors
+# Maintainers 
 
-[Fran Méndez](https://twitter.com/fmvilas) is currently the project director.
+AsyncAPI is actively maintained by:
 
-Current maintainers are [Mike Ralphson](https://twitter.com/PermittedSoc) and [Raisel Melian](https://twitter.com/raiselmelian).
+- [Fran Méndez](https://twitter.com/fmvilas) - Project Director.
+- [Łukasz Górnicki](https://twitter.com/derberq) - Maintainer and Dev Community Keeper.
 
-So far, we have had many contributors and we are thankful for their time and help. Until the day of this writing, main contributors are:
+So far, we have had many contributors and we are thankful for their time and help.
 
-* [Bruno Pedro](https://twitter.com/bpedro)
-* [Chris Wood](https://twitter.com/sensiblewood)
-* [Dulce De La Rosa](https://twitter.com/DeLaR0sa)
-* [Emmelyn Wang](https://twitter.com/lifewingmate)
-* [Jonathan Schabowsky](https://twitter.com/JSchabowsky)
-* [Kin Lane](https://twitter.com/kinlane)
-* [Lukasz Gornicki](https://twitter.com/derberq)
-* [Maciej Urbańczyk](https://github.com/magicmatatjahu)
+# Contributors
 
-Would you like to contribute? Check out our [contributing guidelines](https://github.com/asyncapi/asyncapi/blob/master/CONTRIBUTING.md).
+We do our best to recognize every contribution to the project. We do it individually in every repository from [AsyncAPI GitHub organization](https://github.com/asyncapi/). There should always be a **Contributors** section in the readme, like [this one](https://github.com/asyncapi/asyncapi/#contributors). We use [All Contributors](https://allcontributors.org/docs/en/specification) specification to handle recognitions. 
```

Comment:
```suggestion
We do our best to recognize every contribution to the project. We do it individually in every repository from [AsyncAPI GitHub organization](https://github.com/asyncapi/). There should always be a **Contributors** section in the readme, like [this one](https://github.com/asyncapi/asyncapi/blob/master/README.md#contributors). We use [All Contributors](https://allcontributors.org/docs/en/specification) specification to handle recognitions. 
```

The current link doesn't work well on mobile because READMEs are trimmed.